app/assets/stylesheets/trips.css.scss

```diff
@@ -58,22 +58,22 @@
   border: solid 3px #363636;
 }
 
-.call-to-action {
+.call-to-action, .call-to-action:visited, .call-to-action:link {
   display: block;
-  font-size: 1.2em;
+  font-size: 1.3em;
   color: white;
   background-color: #544e9c;
-  padding: 30px 0 30px 30px;
+  padding: 30px 0 30px 0px;
   text-decoration: none;
   border: solid 1px #363636;
-  border-radius: 5px 5px 5px 5px
+  border-radius: 5px 5px 5px 5px;
```

Comment:
Shorthand form for property `border-radius` should be written more concisely as `5px` instead of `5px 5px 5px 5px`